### ISSUE TITLE
UI: New focused states on inputs looks.... broken, for lack of a better word

### ISSUE THREAD

**Issue by daattali** [rstudio/rstudio#7242]
Version 1.3.959 Windows 10

It seems like one of the recent version of RStudio included UI changes to add "focus states" to many inputs. While that's a very worthy cause, I think it wasn't implemented in a visually appealing way, and I actually thought I was seeing bugs initially. The light blue box that appears around inputs looks out of place and does not flow with the aesthetics very well. This is the first time I've ever had any negative feedback about anything UI related in RStudio - it had to happen eventually :)

To show a few examples:

1. Around the "Next"/"Prev"/"All" buttons in the Find toolbar, the box is not centered

![image](https://user-images.githubusercontent.com/952340/85915575-9cf03280-b816-11ea-8a5e-f2647530d22f.png)

2. The "Go to file/function" input, when you're focused inside it and start typing, the outline makes it look as if it's three entirey different components, the blue outline makes a very sharp cut around the input that confused me and made me take another look at where my cursor is

![image](https://user-images.githubusercontent.com/952340/85915609-fa847f00-b816-11ea-9e2a-b3f30eee626f.png)

3. The commit message has rounded corners. But when you click on it to start typing, the outline around it is not rounded, yet you can still see the gray rounded corners peeking inside

![image](https://user-images.githubusercontent.com/952340/85915593-e5a7eb80-b816-11ea-9768-9f80ac7eb94a.png)

I hope the UI of the focused-state feature will be improved to look more streamlined, or a way to disable it for those who don't need it and feel that it's a bit of a hindrance in its current form would be good too.

**Comment by gtritchie** [comment]
We will gradually improve look of these, it is a very time-consuming exercise given vast amount of UI and variations in the underlying HTML/CSS.

Agree an option to turn this off would be a good idea as well, given it won't be a quick fix to improve the look.

**Comment by gtritchie** [comment]
In the case of the "go to file/function input", it is actually two components. You can separately focus the text field, and the "x" button next to it, so they do need separate focus indicators. But agree the presentation needs improvement, possibly including the cosmetic arrow in the rectangle for the textbox.

**Comment by daattali** [comment]
Regarding the question in the summit message: my suggestion "Always show focus outlines"

**Comment by gtritchie** [comment]
@daattali I was being tongue-in-cheek with that question but your suggestion is a good one and I’ll change it to that.

**Comment by daattali** [comment]
You didn't add a sacrasm tag :(

**Comment by gtritchie** [comment]
No worries. 🙂

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.747, R Version - 4.0.2, Windows 10 x64 - Desktop

The 'Always show focus outlines'option has been added to Global Options > Accessibility. The option appears to be functioning successfully (it does not disable ALL focus rectangles, but I did a general test and the remaining focus elements seem appropriate and not TOO obtrusive when the option is disabled).
